scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/OptimizerEarliestEndTime.java

```diff
@@ -11,7 +11,9 @@
   @Override
   public boolean isBetterThanCurrent(List<ActivityInstance> candidateGoalSolution) {
     ActivityInstance act = ActivityInstance.getActWithEarliestEndTtime(candidateGoalSolution);
-
+    if(act == null || !act.hasEndTime()) {
```

Comment:
Can you help me understand how this change is classified as linting/style clean up? This seems to clarify existing expectations or add new behavior. This belongs in a separate commit so your review is easily able to pick out code clean up from behavior changes.

Reply:
The linter detected that this could be null or not have an end time. So I protected the rest of the call.

Reply:
That's fair. I think these changes lends themselves nicely to being a single commit. The reviewer can infer that the developer was intentional about the very localized nature of the change, making it fast to review.